src/solutions/concepts/kafka-proxies/iot-ingest-control.md

```diff
@@ -18,6 +18,18 @@ You will need to choose a wildcard DNS pattern to use for public internet access
 
 The [Zilla Plus for Amazon MSK](https://aws.amazon.com/marketplace/pp/prodview-jshnzslazfm44) IoT Ingest and Control Broker lets authorized Kafka clients connect, publish messages and subscribe to topics in your Confluent Cloud cluster via the internet.
 
+![MSK IoT Ingest and Control Overview](/iot_ingestion_control.png)
+
+## Key Features
+
+- Enable **IoT clients** to securely connect, publish messages, and subscribe to Kafka topics using **MQTT protocol**.
+- **No configuration** changes required to your MSK cluster to function as MQTT broker.
+- **Custom Wildcard DNS** & Route 53 Hosted Zone Integration.
+- Seamless end-to-end **TLS** handshake.
```

Comment:
No Route53 Hosted Zone needed, right?
There are 2 TLS handshakes, one from client to Zilla Plus and one from Zilla Plus to Amazon MSK, so not sure what this is trying to emphasize.